plugins/woocommerce/src/Internal/Admin/Marketplace.php

```diff
@@ -12,13 +12,20 @@
  */
 class Marketplace {
 
+	const MARKETPLACE_TAB_SLUG = 'woo';
+
 	/**
 	 * Class initialization, to be executed when the class is resolved by the container.
 	 */
 	final public function init() {
 		if ( FeaturesUtil::feature_is_enabled( 'marketplace' ) ) {
 			add_action( 'admin_menu', array( $this, 'register_pages' ), 70 );
 			add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
+
+			// Add a Woo Marketplace link to the plugin install action links.
+			add_filter( 'install_plugins_tabs', array( $this, 'add_woo_plugin_install_action_link' ) );
+			add_action( 'install_plugins_pre_woo', array( $this, 'maybe_open_woo_tab' ) );
+			add_action( 'admin_print_styles-plugin-install.php', array( $this, 'add_plugins_page_styles' ) );
```

Comment:
Feels a bit icky inserting a style block, but I suppose it beats enqueuing a special stylesheet.

Reply:
> Feels a bit icky inserting a style block, but I suppose it beats enqueuing a special stylesheet.

I didn't want to add another file download just for this.
If it grows I would move it to a file.